install/kubernetes/cilium/values.yaml

```diff
@@ -70,6 +70,15 @@ serviceAccounts:
     name: hubble-generate-certs
     annotations: {}
 
+# -- Configure termination grace period for components.
+terminationGracePeriods:
```

Comment:
I feel like this attribute should go to respective components to be closed to all Deployment attributes. Understand that we have ServiceAccount section for all components in top level, but SA seems to be special case.

.Values.terminationGracePeriodSeconds --> agent
.Values.preflight.terminationGracePeriodSeconds -> preflight
.Values.relay.terminationGracePeriodSeconds -> relay